### PR TITLE
Schema generation fixes for credentials and presentations, support fo…

### PR DESCRIPTION
**Pull request by lovesh**
…r schema URI and private status list cred

### CODE REVIEW

**Comment by lovesh** [comment]
Some of these test failures are due to URLs like `https://w3id.org/vc/status-list/2021/v1` not being resolved.

**Review by cykoder — APPROVED** [review]
lgtm, might be good to get @olegnn's review also as it touches some code he produces

File: src/utils/vc/credentials.js [hunk 1]
```diff
@@ -466,22 +472,200 @@ export async function issueCredential(
...
+    if (!verifyResult.verified) {
+      throw new Error(`Status list credential failed to verify with error: ${verifyResult.error}`);
+    }
+  }
+
+  // decode list from SL VC
+  // get JSON StatusList
+  const { credentialSubject: sl } = privateStatusListCredential;
+  const { encodedList } = sl;
+  const list = await decodeList({ encodedList });
```
Comment (olegnn) [inline src/utils/vc/credentials.js:580]: Wouldn't be better to accept `privateStatusListCredential` as an instance of `PrivateStatusList2021Credential` and use `await privateStatusListCredential.decodedStatusList()` instead?

**Review by olegnn — APPROVED** [review]
Great

File: src/utils/vc/credentials.js (hunk 1 above)
Comment (lovesh) [reply src/utils/vc/credentials.js]: Yes. I had missed the caching aspect. Fixed now. Thanks

**Review by cykoder — APPROVED** [review]

**Pull request merged**